examples/afd_multiple_error_thresholds.py

```diff
@@ -0,0 +1,14 @@
+import desbordante
+import pandas as pd
+pyro = desbordante.Pyro()
+df = pd.read_csv('../examples/datasets/iris.csv', sep=',', header=0)
```

Comment:
```suggestion
df = pd.read_csv('datasets/iris.csv', sep=',', header=0)
```
I think all these examples should be copied using `examples/CMakeLists.txt` so that all of them have this same layout.